pkg/volume/local/local_test.go

```diff
@@ -197,8 +243,64 @@ func TestMountUnmount(t *testing.T) {
 	}
 }
 
+// TestMapUnmap tests block map and unmap interfaces.
+func TestMapUnmap(t *testing.T) {
+	tmpDir, plug := getBlockPlugin(t)
+	defer os.RemoveAll(tmpDir)
+
+	pod := &v1.Pod{ObjectMeta: metav1.ObjectMeta{UID: types.UID("poduid")}}
+	volSpec := getTestVolume(false, tmpDir, true /*isBlock*/)
+	mapper, err := plug.NewBlockVolumeMapper(volSpec, pod, volume.VolumeOptions{})
+	if err != nil {
+		t.Errorf("Failed to make a new Mounter: %v", err)
+	}
+	if mapper == nil {
+		t.Fatalf("Got a nil Mounter")
+	}
+
+	expectedGlobalPath := path.Join(tmpDir, testGlobalPath)
+	globalPath, err := mapper.GetGlobalMapPath(volSpec)
+	if err != nil {
+		t.Errorf("Failed to get global path: %v", err)
+	}
+	if globalPath != expectedGlobalPath {
+		t.Errorf("Got unexpected path: %s, expected %s", globalPath, expectedGlobalPath)
+	}
+	expectedPodPath := path.Join(tmpDir, testPodPath)
+	podPath, volName := mapper.GetPodDeviceMapPath()
+	if podPath != expectedPodPath {
+		t.Errorf("Got unexpected pod path: %s, expected %s", podPath, expectedPodPath)
+	}
+	if volName != testPVName {
+		t.Errorf("Got unexpected volNamne: %s, expected %s", volName, testPVName)
+	}
+	devPath, err := mapper.SetUpDevice()
+	if err != nil {
+		t.Errorf("Failed to SetUpDevice, err: %v", err)
+	}
+	if _, err := os.Stat(devPath); err != nil {
+		if os.IsNotExist(err) {
+			t.Errorf("SetUpDevice() failed, volume path not created: %s", devPath)
+		} else {
+			t.Errorf("SetUpDevice() failed: %v", err)
+		}
+	}
+
+	unmapper, err := plug.NewBlockVolumeUnmapper(testPVName, pod.UID)
+	if err != nil {
+		t.Errorf("Failed to make a new Unmapper: %v", err)
```

Comment:
Fatalf (though checking unmapper == nil below can guard that too)

Reply:
Fixed.